README.md

```diff
@@ -63,3 +63,38 @@ To read about aspects of Reactive Trader in greater depth, check out the followi
 
 Reactive Trader was written by the team at [Adaptive](http://weareadaptive.com/), a consultancy that specialises in building real-time trading systems. We have many years of experience in building trading systems for clients with highly demanding latency and reliability requirements. Over the years we have learnt quite a few lessons, and wanted to talk about and point to examples of how we solve technical problems related to real-time delivery of messages.
 
+## Deployment
+
+We practice _continous delivery. Every merge to master triggers a build and publishing of the application's images.
+The commit will be tagged with the latest semver version (master by default is a new minor version), the published
+docker images will be similarly tagged with the same version, and there will  be a version number published in
+the application's UI. The new version will be automatically deployed to the [`dev`](https://web-dev.adaptivecluster.com/) environment.
+
+![Master deployment](./docs/images/deploy-master.png "Master deployment")
+
+To get a version into another environment, tag the version with `env-<environment_name>` e.g. `env-demo`.
```

Comment:
Please include cmd-line instructions for doing this, and how to do with the GitHub website as well.

Reply:
Done :)